packages/jest-config/src/normalize.js

```diff
@@ -571,6 +571,16 @@ export default function normalize(options: InitialOptions, argv: Argv) {
     );
   }
 
+  for (const key of [
```

Comment:
Does it make sense to extract these special options? https://github.com/facebook/jest/pull/5476/files#diff-a6aea6252dbe0924a79aabb9b1fb581dL23